app/models/spree/package_decorator.rb

```diff
@@ -17,15 +14,55 @@ def build_sku_list
         contents.map { |item| item.variant.sku }.join("|")[0..35] # Most carriers have a 35 char limit
       end
 
+      def ref_number
+        order.number
+      end
+
+      def customs_required?
+       shipping_address = order.ship_address
+       (stock_location.country != shipping_address.country) || (shipping_address.state.name.include? "Armed Forces")
+      end
+
+      def easypost_customs_info
```

Comment:
use `.map` instead of `.each` for performance and clarity
```
def easypost_customs_info
        return if !customs_required?

        customs_items = contents.map do |item|
          variant = item.variant
          product = variant.product

          ::EasyPost::CustomsItem.create(
            description: product.taxons.map { |taxon| taxon.name }.join(" "),
            quantity: item.quantity,
            value: variant.price * item.quantity,
            weight: variant.weight,
            hs_tariff_number: product.easy_post_hs_tariff_number,
            origin_country: stock_location.country.try(:iso),
          )
        end
end
```